src/app/window-handler.ts

```diff
@@ -305,6 +306,7 @@ export class WindowHandler {
     this.mainWindow = new BrowserWindow({
       ...this.windowOpts,
       ...getBounds(this.config.mainWinPos, DEFAULT_WIDTH, DEFAULT_HEIGHT),
+      icon: nativeImage.createFromPath('images/icon-2.ico'),
```

Comment:
I didn't see icon-2 either images folder. Was this an electron pre-defined arguments?